book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -8,24 +8,324 @@ This can involve hosting different events that are attended by people with lived
+
+When planning the dates for the event, it is also useful to consider the potential travel times of your attendees. 
+If your event takes place in person, try to choose a date and start time that takes into account how long your attendees will need to travel. 
+For example, having your event start at 10:30 a.m. allows for inter-city travel the morning before the event. 
+
+(cl-inclusive-events-timeline-planning)=
+#### Creating a Planning Timeline
+Your planning timeline will vary greatly based on the scale of your event. 
+We have highlighted areas where your timeline should take into account any considerations that would be specifically supportive of people with lived experience.
+
+Although it is always necessary to allow for appropriate time to source speakers, people with lived experience may require more notice than is traditionally planned for in order to support their attendance and speak at an event. 
+People with lived experience may have additional constraints such as medical appointments, caring responsibilities, support and travel needs, and preparation time which can require significant time in order to plan for and implement. 
+Try to identify potential speakers as early as possible and book dates in their diaries that would allow them to make plans that would support their engagement in the event.
+
+Where possible, try to arrange a recording of a talk or an alternative session plan that can be used in the event a speaker or facilitator is unable to attend. 
+You can also arrange to have speakers join remotely if it would be more supportive of their engagement. 
+
+`````{admonition} Case Study
+:class: tip
+The AI for Multiple Long Term Conditions Conference in April 2023 had a keynote speaker join and give their keynote address remotely in order to best support their involvement. 
+People with the lived experience of multiple long-term conditions may find travelling to events taxing which can have a negative impact on their energy levels and ability to engage in daily activities. 
+By supporting the speaker to speak remotely, they were able to give an engaging and thoughtful keynote address without the burden of having to travel. 
+`````
+
+(cl-inclusive-events-planning-GDPR)=
+### GDPR and Data Protection
+Depending on your location and organization, the collection of data during registration may be regulated and need additional considerations.
+
+In the EU, GDPR (or the UK's version UK-GDPR) considers data concerning health conditions and disability as "special category data". 
+This "special category" also extends to cover the data about a relative or carer of an individual with a health condition or disability. 
+Depending on the specific attendees and focus of your event, you may need to complete additional data protection forms and plan for how to manage the protected information. 
```

Comment:
Depending on the specific attendees and focus of your event, you may need to complete additional data protection forms and establish a plan for managing the protected information.
```suggestion
Depending on the specific attendees and focus of your event, you may need to complete additional data protection forms and plan for how to manage the protected information. 
```

Reply:
Updated :)